controllers/app.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectAllTopics } = require("../models/app.model");
+
+exports.getAllTopics = (req, res, next) => {
+  selectAllTopics()
+    .then(({ rows }) => {
+      const topics = rows;
```

Comment:
Anything database related should really be handled in the model. If you return the rows from your model, you should only need to call selectAllTopics here, then send the response. 👍

Reply:
updated